backend/migrations/20230912181446_init.up.sql

```diff
@@ -0,0 +1,28 @@
+
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+ 
+CREATE TABLE attestation_requests (
```

Comment:
Better do CREATE TABLE IF NOT EXISTS, so this stays idempotent, even though it is not supposed to be executed two times

Reply:
updated in: [e481678](https://github.com/KILTprotocol/attester/pull/1/commits/e481678e3283b8a9a1c8ba5c1ee0e133ef0ecc6a)